app/config.go

```diff
@@ -70,6 +71,20 @@ type Config struct {
 	FacebookOauthCredentials    *oauth.Credentials
 }
 
+// Lazy initialize Logger for code using partially initialized Config
+func (c *Config) GetLogger() logrus.FieldLogger {
+	if c.Logger == nil {
```

Comment:
if `GetLogger` sticks around, this could be a good case for [sync.Once](https://godoc.org/sync#Once).

Reply:
Yes I suppose there could be a race here. In the current code I'm fairly certain there is not since `GetLogger` gets called early in initialisation before we have spun off goroutines. I might refactor it away otherwise why not.